frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -780,7 +780,8 @@ const QueueItem = ({
                 id="patient-name-header"
               >
                 <div className="card-header">
-                  <div
+                  <Button
```

Comment:
I don't have screenshots handy, but FYI @DanielSass and I inspected the box model and display properties for this change and we can confirm that the layout is intact down to the pixel 🕵️